Ryan Bullough/week01/Wednesday/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wednesday 16 September</title>
+    <script src="main.js"></script>
+  </head>
+  <body>
+    <h1>Wednesday 16 September</h1>
+  </body>
+</html>
```

Comment:
I feel like you discovered the shortcut for creating this boilerplate - nice work!